javascript/tree-js/index.js

```diff
@@ -0,0 +1,37 @@
+function tree(nameItemsObj, indent) {
+  for (const [key, value] of Object.entries(nameItemsObj)) {
+    // Print out name
+    if (key === 'name') {
+      console.log(indent, value)
+    }
+
+    // Indent items
+    if ((key === 'items') & Array.isArray(value)) {
+      // let indentItem = ' ├——',
+      //   indentChild = ' │   └——'
+      let indentItem = ' \u251c\u2014\u2014',
+        indentChild = ' \u2502   \u2514\u2014\u2014'
+      indent = indent === indentItem ? indentChild : indentItem
+
+      value.map((item) => {
+        tree(item, indent)
+      })
+    }
+  }
+}
+
+const makeTreeObj = {
```

Comment:
<img width="640" alt="Screenshot 2020-08-15 at 17 16 54" src="https://user-images.githubusercontent.com/1259644/90315380-5916d100-df1b-11ea-9229-e4099e0a36b1.png">

It does the thing with this object. However, I can still see a room for improvement. Like the last child items of the root might not have the branches out of them.